docs/nx/azure-react/plugin-information.md

```diff
@@ -0,0 +1,65 @@
+---
+id: ensono-stacks-azure-react
+title: "@ensono-stacks/azure-react"
+sidebar_label: "@ensono-stacks/azure-react"
+description: "Creates a ReactJS application and configures the ReactJs application for Azure!"
+keywords:
+  - Nx
+  - monorepo
+  - stacks
+  - ensono
+  - azure-react
+  - generator
+  - executor
+  - appinsights
+---
+
+import GeneratorAppInsights from './generator-app-insights-web.md';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Creates a ReactJS application and configures your ReactJS application to use various Azure services:
```

Comment:
Does it create a ReactJS app? I thought it just enhances an existing one with our additions?

Reply:
It does it runs the NX react generator

`import { libraryGenerator } from '@nrwl/react';`

`  // Use the existing library generator
    await libraryGenerator(tree, {
        ...options,
        linter: Linter.EsLint,
        style: 'none',
    });`

Reply:
Ah I misunderstood the code. It actually does neither. It creates a ReactJS library which can then be imported into an existing react app. Updated the docs to reflect this